### PR TITLE
Add report for the RustDay MX 2017 event

### PR DESCRIPTION
**Pull request by sebasmagri**

### CODE REVIEW

File: _posts/2017-07-23-rustdaymx-2017.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "Making of RustDay Mexico City 2017, a RustBridge day."
+author: Sebastián Magrí
+description: "The makings of the very first RustBridge event in Mexico City"
+---
+
+On June 17th, the [Rust-MX](https://github.com/rustmx/team/) group teamed with [Mozilla México](http://www.mozilla-mexico.org/) to do its very first _whole day_ *Rust* event in Mexico City at the [Nearsoft](https://nearsoft.com/) facilities, the [RustDay 2017](https://2017.rustday.mx/).
```
Comment (badboy) [inline _posts/2017-07-23-rustdaymx-2017.md]: `teamed up with`

File: _posts/2017-07-23-rustdaymx-2017.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+After [5 meetups](https://meetup.com/rust-mx), the team decided it was a good moment to go a step further and plan a workshops day using the [RustBridge](https://bridge.community.rs/) curriculum as a starting point and focusing on reaching out to underrepresented groups in the city. The initiative quickly gained support across communities and companies in the city. Many thanks to _Mozilla México_, _Nearsoft_, [Software Gurú](https://sg.com.mx/) and [SocialTIC](https://socialtic.org/) for their support.
```
Comment (badboy) [inline _posts/2017-07-23-rustdaymx-2017.md]: `After 5 previous meetups`

Comment (badboy) [reply _posts/2017-07-23-rustdaymx-2017.md]: `workshop day` (without the s on workshop)

File: _posts/2017-07-23-rustdaymx-2017.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+### The CoC focal points
+
+Two members of _SocialTIC_ installed a table near the entrance offering information on digital security and inclusiveness, but they were also designed as the CoC focal points. If any person had any concern related to the [Code of Conduct](https://2017.rustday.mx/CoC.html) or had a claim to do about a specific situation during the day, the CoC focal points were going to handle it. Many thanks to _SocialTIC_ for the help with this!.
+
+### The mentors
+
+Formed by members of the _Rust-MX_ group and with the help of the [Rust Community](https://community.rs) members _Brian Anderson_ and _Erick Tryzelaar_, the mentors team was in charge of preparing the workshops materials, _training the trainers_, and of providing guidance to the attendees during the day. We wanted to make sure each table had at least one mentor available to help out people so they did not get stuck and to make sure they were able to understand the basic concepts we were trying to demonstrate in the workshops.
```
Comment (badboy) [inline _posts/2017-07-23-rustdaymx-2017.md]: `and of` -> `and for`

File: _posts/2017-07-23-rustdaymx-2017.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+#### The mentors preparation
+
+The workshops materials were decided and prepared during three sessions before the event.
+
+The first session was done to define the goals of the event, so we decided to run a small survey within the people who had reserved a place for the event until then to gather information about their setup and their interests.
+
+Based on the results of the survey, we met again to review the [RustBridge materials](https://community.rs/rustbridge/) and see how could we adapt it to make them more attractive to the attendees, and to define a final agenda for the event. In the end, we decided to do a very simple _Hello world_ like tutorial, a simplified [Web Scraper](https://community.rs/rustbridge/web-scraper/README.html) workshop using the `reqwest` crate instead of `hyper`, this time to scrap cryptocoin prices, and a simplified [Text Adventure](https://community.rs/rustbridge/textadventure/textadventure.html). Additionally, we slotted time for a keynote, a talk and a couple lighting talks for local people to share their experiences with Rust.
```
Comment (badboy) [inline _posts/2017-07-23-rustdaymx-2017.md]: `how could we adapt` -> `how we could adapt`

Comment (badboy) [reply _posts/2017-07-23-rustdaymx-2017.md]: `scrap` -> `scrape`

File: _posts/2017-07-23-rustdaymx-2017.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+Finally, a third session was held to go through the _Web Scraper_ and _Text Adventure_ modifications, push it to [GitHub](https://github.com/rustmx) so it was available for people to fetch it, and make sure the mentors were prepared to guide the attendees.
+
+## The day of the event
+
+The event day arrived and everyone was really anxious about it. We arrived the venue early to set up everything. People started coming and the _Installers_ team was there to help them. _Brian_ and _Erick_ were also helping everyone have their environments set up.
```
Comment (badboy) [inline _posts/2017-07-23-rustdaymx-2017.md]: `arrived at the venue`

File: _posts/2017-07-23-rustdaymx-2017.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+Next, _Erich Córdoba_ talked about [How to convince your boss to use Rust](https://youtu.be/1EFioxmCcic), and after that we had some [delicious basket tacos](https://flic.kr/p/VX7qYr).
+
+After the lunch, _Abiel Parra_ did the second workshop to show the attendees how to [Implement a Cryptocoin Prices Scraper using Rust](https://youtu.be/APBJmZ-dRbA), so they could start going a little bit deeper into the Rust syntax and learn how to use crates.
+
+To finish, _Sebastián Magrí_ demonstrated how to [Build a Text Adventure](https://youtu.be/i1wU70E1yS8), focused on explaining project organization, data structures and lifetimes.
+
+## Wrap up
+
+In the end, the team and the attendants were really satisfied. There were no CoC related issues and we got a peak of 52 participants that, plus volunteers and staff, did a total number of 72 persons.
```
Comment (badboy) [inline _posts/2017-07-23-rustdaymx-2017.md]: `attendants` -> `attendees`

File: _posts/2017-07-23-rustdaymx-2017.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+The code of the workshops is available at [Rust-MX's GitHub](https://github.com/rustmx), and 3 weeks later, the team had the opportunity to follow up with two workshops at [TelmexHub](). The next meetup is set to happen on July 26th, and we're already planning another whole day event for November or December. We had a lot of lessons learned from this first time, which we look forward to apply to the next one.
+
+We hope this experience is useful for people in other places to do their own workshop days. Please, get in touch with [us](mailto:community-team@rust-lang.org) if you need support or feedback. We look forward to work with you to bring more Rust to more people.
```
Comment (badboy) [inline _posts/2017-07-23-rustdaymx-2017.md]: Put `get in touch with us` into the mailto link (instead of just the single word). Much easier to hit then.

**Review by badboy — CHANGES_REQUESTED** [review]
I noted some small things, but otherwise good to go!

**Comment by sebasmagri** [comment]
Thanks @badboy, I've updated the post with your corrections. Regards!

**Review by badboy — APPROVED** [review]

**Comment by sebasmagri** [comment]
What would be needed to merge this? :-)

**Comment by badboy** [comment]
Someone hitting the merge button!

**Comment by badboy** [comment]
Which I just did

**Comment by badboy** [comment]
Feel free to post to URLO, Reddit and tweet it mentioning @rustlang

**Comment by sebasmagri** [comment]
Thanks @badboy!

**Pull request merged**